undertow/src/main/java/org/wildfly/extension/undertow/security/LogoutNotificationReceiver.java

```diff
@@ -41,22 +40,34 @@
 public class LogoutNotificationReceiver implements NotificationReceiver {
 
     private final AuthenticationManager manager;
+    private final String securityDomain;
 
-    public LogoutNotificationReceiver(AuthenticationManager manager) {
+    public LogoutNotificationReceiver(AuthenticationManager manager, String securityDomain) {
```

Comment:
I don't think it really matters much either way, the end result should be the same.